docs/pages/eas-insights/introduction.mdx

```diff
@@ -6,7 +6,7 @@ description: An introduction to EAS Insights which is a preview service for proj
 
 import { Terminal } from '~/ui/components/Snippet';
 
-> **info** EAS Insights is in preview and subject to breaking changes. While in preview, it is free to use.
+> **important** **EAS Insights** is in preview and subject to breaking changes. While in preview, it is free to use.
```

Comment:
Can you add this new callout type and its use case description to docs/Readme > Use Callout section?

This helps both internally and externally when seeking info for what type is expected to be used when.